bp_common/syn/Makefile.vcs

```diff
@@ -72,7 +72,7 @@ $(BUILD_DIR)/simv: | $(BUILD_COLLATERAL)
 	-@tail -n3 $(BUILD_LOG) > $(BUILD_REPORT)
 	-@test -s $(BUILD_ERROR) && echo "FAILED" >> $(BUILD_REPORT) || rm $(BUILD_ERROR)
 
-build_dump.v: VCS_BUILD_OPTS += -debug_pp
+build_dump.v: VCS_BUILD_OPTS += -debug_all
```

Comment:
TODO: revert once ready to merge

(Also, why isn't this the default? Do we _like_ garbage in our debug traces?)

Reply:
:) TL;DR big dump files for big projects. Haven't benchmarked it for BP though...